src/pages/Components/Select/specs.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 'Specs'
+order: 3
+---
+
+## Anatomy
+
+![selects-documentation](Selects-documentation.png)
+
+## Properties
+
+### CSS Tokens : layout
+
+| State       | Property      | Token                |
+| ----------- | ------------- | -------------------- |
+| **Default** | padding       | `mu-100` (1rem)      |
+| **Default** | border-radius | `radius.m` (4px)     |
+| **Default** | border-width  | `border.s` (1px)     |
+| **Default** | font-size     | `size.font.05`(16px) |
+
+### CSS Tokens : state colors
+
+> **You must always use the tokens, the colors are only there for that information.**
```

Comment:
You must always use the tokens, the colors are listed here for information purposes only